tools/deadlock.py

```diff
@@ -465,7 +475,10 @@ def main():
             print('%s. Is the process (pid=%d) running?' % (str(e), args.pid))
             sys.exit(1)
 
-    bpf = BPF(src_file=b'deadlock.c')
+    text = open('deadlock.c').read()
```

Comment:
It would be better to use something like `with open('deadlock', 'r') as f:` :)

Reply:
adjusted.